### DISCUSSION TITLE
Apache Won't start after configuring a Non Thread Safe version of PHP

### DISCUSSION THREAD

**Discussion by Pantheroso** [leokhoa/laragon#628] [Q&A]
Hello everyone, I am using version 6 of Laragon. Apache version: 2.4.58, MySQl version: 8.2.0 and PHP: 8.1.26. After configuring Laragon to use the NTS version of Php, I got an error message at startup, Apache is not launching.:
Service Apache can not start. Reason:
--------------------------------------------
httpd: Syntax error on line 547 of 
D:/*laragon/bin/apache/httpd-2.4.58-win64-VS17/co...: 
Syntax error on line 1 of D:/*laragon/etc/apache2/fcgid.conf: Cannot load 
D:/#laragon/etc/apache2/modules/mod_fcgid-2.3.9-... into server: Le module sp\xe9cifi\xe9 est introuvable. 

What can I do to fix it please?

**Comment by vahidirp** [comment]
It looks like there is an issue with the mod_fcgid module in your Apache configuration. The error message indicates that Apache is unable to load the mod_fcgid module because it cannot find the specified module file. Here are a few steps you can take to troubleshoot and resolve this issue:

    Check if the mod_fcgid module file exists:
        Navigate to the specified directory: D:/#laragon/etc/apache2/modules/.
        Verify if the mod_fcgid-2.3.9-Win64-VS17.so file (or a similar file) is present.
        If the file is missing, you may need to download and install the missing module file or reconfigure Apache to use an existing one.

    Download the correct mod_fcgid module:
        Ensure that you are using the appropriate version of the mod_fcgid module that is compatible with your Apache version.
        You can download the module from a reliable source and place it in the specified directory.

    Update Apache configuration:
        Open the D:/*laragon/etc/apache2/fcgid.conf file and verify its contents.
        Ensure that the LoadModule directive for mod_fcgid is correctly pointing to the module file. It should look something like this:


```
LoadModule fcgid_module modules/mod_fcgid-2.3.9-Win64-VS17.so
```
        If the file path is incorrect, update it to point to the correct location of the mod_fcgid module.

    Restart Laragon:
        After making changes to the configuration files, restart Laragon to apply the changes.

If the issue persists, double-check your configuration files for typos or errors, and consider seeking assistance from the Laragon community or forums for more specific guidance based on your Laragon version. Additionally, ensure that your PHP and Apache versions are compatible with each other.

**Reply by Pantheroso** [reply]
Hello, Thank you very much for this effective solution. Nice weekend.
